### PR TITLE
Rename Ansible guide

### PR DESCRIPTION
**Pull request by Lennonka**
Configuring and using Ansible integration to configure hosts in Project is analogic to the Puppet integration.
I suggest modifying the title of the guide to make it more obvious.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/attributes-titles.adoc [hunk 1]
```diff
@@ -13,6 +12,7 @@
 // Installing Disconnected - defined for Satellite only
 :InstallingServerDisconnectedDocTitle: Installing {ProjectServerTitle} in a Disconnected Network Environment
 :InstallingSmartProxyDocTitle: Installing an External {SmartProxyServerTitle} {ProjectVersion}
+:ManagingConfigurationsAnsibleDocTitle: Configuring Hosts Using Ansible
```
Comment (maximiliankolb) [inline guides/common/attributes-titles.adoc:15]: 👍

**Comment by Lennonka** [comment]
@ares Yay or nay? :)

File: guides/doc-Managing_Configurations_Ansible/docinfo.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-<title>Configuring Red Hat Satellite to use Ansible</title>
+<title>Managing Configurations Using Ansible Integration in Red Hat Satellite</title>
 <productname>Red Hat Satellite</productname>
 <productnumber>6.12</productnumber>
-<subtitle>Configure Ansible in Satellite to help automate repetitive tasks</subtitle>
+<subtitle>Use Ansible integration in Satellite to configure host systems</subtitle>
 <abstract>
-    <para>This guide describes how to set up and configure Red Hat Satellite to use Ansible to perform remote execution and automate repetitive tasks.</para>
+    <para>This guide describes how to integrate Red Hat Satellite with Ansible and how to perform remote execution and automate repetitive tasks using Ansible roles and playbooks in Satellite.</para>
```
Comment (maximiliankolb) [inline guides/doc-Managing_Configurations_Ansible/docinfo.xml:6]: I am unsure if foreman_ansible can actually run Ansible Playbooks. In docs, I've only found how to import playbooks into Foreman: https://docs.theforeman.org/nightly/Configuring_Ansible/index-katello.html#importing-an-ansible-playbook-by-name_ansible

**Review by maximiliankolb — APPROVED** [review]
LGTM!

File: guides/doc-Managing_Configurations_Ansible/docinfo.xml (hunk 2 above)
Comment (Lennonka) [reply guides/doc-Managing_Configurations_Ansible/docinfo.xml:6]: Ansible playbooks can be used in job templates and run in REX jobs, that's why we have a module on importing them. I think that we're not explicit enough about it in the REX chapter, though.
image

**Comment by Lennonka** [comment]
No protests, I'm merging this. :)

**Comment by Lennonka** [comment]
Cherry-picked:
- 8ef610a90..bc38e724b  3.5 -> 3.5
- 5dfa46429..397c0d0e4  3.4 -> 3.4
- adf53a7d1..b6fb61a4e  3.3 -> 3.3

**Comment by ekohl** [comment]
When renaming a guide, please also open a PR to purge the old content from the `gh-pages` branch. Otherwise users may consume outdated information. Ideally speaking we'd have a redirect, but for now we can live with a 404.

**Comment by maximiliankolb** [comment]
> When renaming a guide, please also open a PR to purge the old content from the `gh-pages` branch. Otherwise users may consume outdated information. Ideally speaking we'd have a redirect, but for now we can live with a 404.

cc @Lennonka 

Yes, but a user experience point of view, we should definitely do this!

Is this as simple as branching gh-pages and running "git rm -r SOME DIRECTORY"?

**Comment by ekohl** [comment]
Yes, it's pretty much that.

**Pull request merged**